### PR TITLE
Replace references to 'master/slave' in the spec doc

### PR DESCRIPTION
**Pull request by bocon13**
Using the following mapping:
    - master -> primary
    - slave -> backup
    - mastership arbitration -> client arbitration
    - mastership -> primary client

**TODO**
- [x] clean up line lengths
- [x] update revision history

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk
Comment (bocon13) [inline docs/v1/P4Runtime-Spec.mdk:378]: This might not quite be right.

**Comment by antoninbas** [comment]
As an additional TODO, some figures in the spec need to be updated

File: docs/v1/P4Runtime-Spec.mdk
Comment (antoninbas) [reply docs/v1/P4Runtime-Spec.mdk:378]: I feel like "multi-master" protocol was not right to begin with. I am not sure what it's supposed to mean here. The figure shows a single primary controller. Maybe we should just replace "A multi-master protocol allows more than one controller to participate" with "More than one controller can participate".

File: docs/v1/P4Runtime-Spec.mdk
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk:610]: Same as above. I don't think "multi-master" was accurate in the first place, and I am not sure "active-active" applies here since the backup controllers don't do anything until the primary fails. So I would suggest simply "the P4Runtime client arbitration scheme"

File: docs/v1/P4Runtime-Spec.mdk
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk:5337]: s/primarys/primaries

or "before selecting a different primary"

**Review by antoninbas — COMMENTED** [review]
I think the "active-active" does not really apply, otherwise changes LGTM. Some figures also need to be updated.

File: docs/v1/P4Runtime-Spec.mdk
Comment (bocon13) [reply docs/v1/P4Runtime-Spec.mdk:378]: How about this?

"P4Runtime only allows for a single primary controller to actively write to each read/write entity. P4 entities can be divided into different groups, called roles. P4Runtime allows for a primary controller for each role, and a role-based arbitration scheme..."

File: docs/v1/P4Runtime-Spec.mdk
Comment (bocon13) [reply docs/v1/P4Runtime-Spec.mdk:631]: That make sense, we can also do: "but the P4Runtime role-based client arbitration scheme"

File: docs/v1/P4Runtime-Spec.mdk
Comment (bocon13) [reply docs/v1/P4Runtime-Spec.mdk:5358]: I like the latter better.

File: docs/v1/P4Runtime-Spec.mdk
Comment (bocon13) [inline docs/v1/P4Runtime-Spec.mdk:360]:
```suggestion
    which is used for packet I/O and client arbitration, among other
```

**Review by antoninbas — COMMENTED** [review]
Apart from one typo, this looks good to me. In the future, please consider creating a branch directly in this repo since you have write access (it lets us look at the rendered spec HTML at `https://s3-us-west-2.amazonaws.com/p4runtime/travis/<your_branch_name>/P4Runtime-Spec.html` before merging the PR, in order to catch formatting issues. I generated the docs manually on my laptop and everything looks fine, including the 2 modified figures.

**Comment by bocon13** [comment]
@antoninbas I made the one fix for the typo that you caught and rebased against latest master. Unfortunately, I don't think that I have write access to the repo which is why I was using my personal branch.

```ERROR: Permission to p4lang/p4runtime.git denied to bocon13.```

**Review by antoninbas — APPROVED** [review]
LGTM

**Comment by antoninbas** [comment]
@bocon13 My bad I thought you already had write access, I sent you an invite

**Comment by antoninbas** [comment]
Merging this PR as it has been open for a while, and there was no objection at the 10/23/2020 WG meeting.

**Comment by bocon13** [comment]
Great, thanks @antoninbas! Looks like write access is working, too.

**Pull request merged**
